external/stdk/tizenrt-wrapper/os/event_groups.h

```diff
@@ -15,6 +15,33 @@
  * limitations under the License.
  *
  ******************************************************************/
+/*
```

Comment:
how about keep consistent with the license in  event_groups.c ?

Reply:
OSV tools gives the code matches with event_groups.h in freertos, so I copy the license information from it. 
Actually, event_groups.c in freertos has the same license, but not the BSD-3 license in out event_groups.c.

Reply:
let's stick with OSV tools.
@sunghan-chang   let's merge this missing license